vendor/knative.dev/operator/pkg/reconciler/common/workload_override.go

```diff
@@ -223,7 +223,7 @@ func replaceProbes(override *base.WorkloadOverride, ps *corev1.PodTemplateSpec)
 				}
 				if *overrideProbe == (v1.Probe{}) {
 					//  Disable probe when users explicitly set the empty overrideProbe.
-					containers[i].ReadinessProbe = nil
+					containers[i].LivenessProbe = nil
```

Comment:
Probably we should have backported this https://github.com/knative/operator/pull/1823

Reply:
we could still, but than... the next "update dependency" has the change

Reply:
I guess you mean in 1.34 ?